tests/acceptance/ManageWelcomeEmailCest.php

```diff
@@ -8,154 +8,141 @@
 
 class ManageWelcomeEmailCest {
 
-  private $welcome_template;
-  private $title_element;
+  private $welcomeTemplate;
+  private $titleElement;
 
   function __construct() {
-    $this->welcome_template = '[data-automation-id="select_template_0"]';
-    $this->title_element = '[data-automation-id="newsletter_title"]';
+    $this->welcomeTemplate = '[data-automation-id="select_template_0"]';
+    $this->titleElement = '[data-automation-id="newsletter_title"]';
   }
 
-  private function createWelcomeEmailWithTitle(\AcceptanceTester $I, $newsletter_title) {
+  private function createWelcomeEmailWithTitle(\AcceptanceTester $I, $newsletterTitle) {
     $newsletter_factory = new Newsletter();
```

Comment:
Seems like you missed this variable. I think it should be `$newsletterFactory` to follow the same convention.